src/data/use-cases/db-load-guardian-by-email.ts

```diff
@@ -0,0 +1,14 @@
+import { type LoadGuardianByEmail } from '@/domain/use-cases'
+import { type LoadGuardianByEmailRepository } from '../protocols'
+
+export class DbLoadGuardianByEmail implements LoadGuardianByEmail {
+  private readonly loadGuardianByEmailRepository: LoadGuardianByEmailRepository
+
+  constructor (loadGuardianByEmailRepository: LoadGuardianByEmailRepository) {
+    this.loadGuardianByEmailRepository = loadGuardianByEmailRepository
+  }
+
+  async load (email: string): Promise<LoadGuardianByEmail.Result> {
```

Comment:
@SamuelS00 and me hava implemented exactly this logic, we need a align about this code duplications.